talamer/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "arrowParens": "always",
+  "singleQuote": true,
```

Comment:
What does this do? We have single quotes for code and double for html. That allows us to write html in code and not have to escape the quotes.

Reply:
This actually refers to quotes in js. I copied our settings from the lint plugin. The reason for file is for the printwidth settings to apply to html.

I'd like to run prettier through our linter like js for html but it doesn't work.